server/history/update_history_db.py

```diff
@@ -14,27 +14,22 @@
 async def update_history(test_mode=False):
     routes = DEFAULT_ROUTE_IDS + SILVER_ROUTE_IDS
     now = datetime.datetime.now(pytz.utc)
-    eastern = pytz.timezone('US/Eastern')
     postgres_conn = get_history_db_connection()
-    json = await mbta_api.vehicle_data_for_routes(routes, test_mode)
+    [vehicles_to_display, _] = await mbta_api.vehicle_data_for_routes(routes, test_mode)
```

Comment:
Completely of scope for this PR, but fwiw it would be more idiomatic for `vehicle_data_for_routes` to return multiple values, rather than a single array, so this unpacking would look like:

```py
vehicles_to_display, _ = await mbta_api.vehicle_data_for_routes(routes, test_mode)
```

Reply:
This function is wired into the /trains route so I'm going to hold off for now—agreed though!